app/src/main/java/org/sagebionetworks/research/mpower/tracking/fragment/TrackingItemViewHolder.java

```diff
@@ -0,0 +1,101 @@
+package org.sagebionetworks.research.mpower.tracking.fragment;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup.LayoutParams;
+import android.view.ViewGroup.MarginLayoutParams;
+
+import org.sagebionetworks.research.mpower.R;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingItem;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingSection;
+import org.sagebionetworks.research.mpower.tracking.view_model.TrackingActiveTaskViewModel;
+
+public class TrackingItemViewHolder extends RecyclerView.ViewHolder {
+    private SelectionUIFormItemWidget widget;
+    private TrackingItem trackingItem;
+    private boolean selected;
+    private TrackingActiveTaskViewModel<?, ?> viewModel;
+
+    public TrackingItemViewHolder(final SelectionUIFormItemWidget itemView, TrackingActiveTaskViewModel<?, ?> viewModel) {
+        super(itemView);
+        this.viewModel = viewModel;
```

Comment:
This is totally a subjective thing, but I find the "this." in front of everything makes the code less readable.

Reply:
I can stop putting this in if everyone prefers. It is a UW style guide thing that they are super serious about so it has been drilled into me to the point where I don't even think about it anymore.